src/plugins/dam/components/image-asset-picker/index.jsx

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import noop from 'lodash/noop';
 
 import { Button } from '@triniti/admin-ui-plugin/components';
-import AssetPickerModal from '@triniti/cms/plugins/dam/components/asset-picker-modal';
+import ImageAssetPickerModal from '@triniti/cms/plugins/dam/components/image-asset-picker-modal';
```

Comment:
Remove the keyword `export` on Ln12. It is not necessary, leaving it will only create linter errors when importing the component. 
https://github.com/triniti/cms-js/pull/179/files#diff-cef6adb279add88f6eeecf2374a4e573R12